web/new_alt_text_manager.js

```diff
@@ -240,29 +240,13 @@ class NewAltTextManager {
 
     let hasError = false;
     try {
-      const { width, height, data } = this.#imageData;
-
-      // Take a reference on the current editor, as it can be set to null (if
-      // the dialog is closed before the end of the guess).
-      // But in case we've an alt-text, we want to set it on the editor.
-      const editor = this.#currentEditor;
-
       // When calling #mlGuessAltText we don't wait for it, so we must take care
       // that the alt text dialog can have been closed before the response is.
-      const response = await this.#uiManager.mlGuess({
-        name: "altText",
-        request: {
-          data,
-          width,
-          height,
-          channels: data.length / (width * height),
-        },
-      });
-      if (!response || response.error || !response.output) {
-        throw new Error("No valid response from the AI service.");
-      }
-      const altText = (this.#guessedAltText = response.output);
-      await editor.setGuessedAltText(altText);
+
+      const altText = await this.#currentEditor.mlGuessAltText(
```

Comment:
Previously we'd also update `this.#guessedAltText`, [see here](https://github.com/mozilla/pdf.js/pull/18541/files#diff-6dbe68273ba0858959b04fe0ca8097d9cf9c8a9fccc75dfc8ef222fbcf12ac07L264), however that's removed in the patch; was that intentional?